build_linux.sh

```diff
@@ -4,12 +4,20 @@
 #
 
 sudo apt-get install patchelf -y
+sudo apt install gcc
```

Comment:
```suggestion
sudo apt-get install -y gcc
```

Missing `-y` and changed apt to apt-get for consistency.